code/osapi/outwnd.cpp

```diff
@@ -69,30 +86,18 @@ void load_filter_info()
 
 	if (!fp) {
 		Outwnd_no_filter_file = 1;
-
-		strcpy_s( new_filter.name, "error" );
-		new_filter.enabled = true;
-		filter_vector().push_back( new_filter );
-
-		strcpy_s( new_filter.name, "general" );
-		new_filter.enabled = true;
-		filter_vector().push_back( new_filter );
-
-		strcpy_s( new_filter.name, "warning" );
-		new_filter.enabled = true;
-		filter_vector().push_back( new_filter );
-
 		return;
 	}
 
 	Outwnd_no_filter_file = 0;
 
 	while ( fgets(inbuf, NAME_LENGTH+3, fp) ) {
+		bool enabled;
```

Comment:
Although I think the `continue` statement below would take care of it, valgrind might throw a warning about this variable not being initialized. It might be nice to just initialize this to `false` and not have to worry about it again.

But valgrind *should* be smart enough to not flag this, so I'll leave it up to you whether or not to make the change.

Reply:
The if() block below looks "neater" with the variable the way it is, so I prefer it as is.  I'll give valgrind the benefit of the doubt, and cross that bridge if we come to it.